specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -2233,6 +2239,24 @@
         "caption": {
           "description": "The caption for the media object, if supported and provided",
           "type": "string"
+        },
+        "animated": {
+          "description": "Set to  true  if the sticker is animated; false otherwise.",
+          "type": "boolean"
+        }
+      }
+    },
+    "AcsMessageReactionContent": {
+      "description": "Message Reaction Content",
+      "type": "object",
+      "properties": {
+        "messageId": {
+          "description": "WhatsApp message ID of message you want to apply the emoji to.",
+          "type": "string"
+        },
+        "emoji": {
```

Comment:
If these properties are required can we make these required here as well in the swagger? The default in swagger is to make everything optional.

Reply:
Hi Libba, 
Made them required in the swagger. Please review and let me know.